week-2/implement/vat.js

```diff
@@ -8,3 +8,9 @@
 // Given a number,
 // When I call this function with a number
 // Then it returns the new price with VAT added on
+
+function vat(number) {
+  return (number * 1.2).toFixed(2);
```

Comment:
Good use of toFixed() - it's not asked in the requirements, but you have correctly identified that money is usually represented with 2 decimal spaces. Well done